packages/typeorm/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@loopback/typeorm",
+  "version": "0.0.1",
+  "description": "Adds support for TypeORM in LoopBack",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "acceptance": "lb-mocha \"dist/__tests__/acceptance/**/*.js\"",
+    "build": "lb-tsc",
+    "clean": "lb-clean loopback-typeorm*.tgz dist *.tsbuildinfo package",
+    "pretest": "npm run build",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "verify": "npm pack && tar xf loopback-typeorm*.tgz && tree package && npm run clean"
+  },
+  "author": "IBM Corp.",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@loopback/boot": "^2.3.5",
+    "@loopback/build": "^5.4.3",
+    "@loopback/eslint-config": "^8.0.1",
+    "@loopback/repository": "^2.9.0",
+    "@loopback/rest": "^5.2.0",
+    "@loopback/service-proxy": "2.3.4",
+    "@loopback/testlab": "^3.1.7",
+    "@types/json-schema": "^7.0.5",
+    "@types/node": "^10.17.26",
+    "sqlite3": "^4.2.0"
+  },
+  "dependencies": {
+    "@loopback/core": "^2.4.1",
+    "@loopback/openapi-v3": "3.4.4",
```

Comment:
Do we need it as a regular dependency?

Reply:
Yes, in https://github.com/strongloop/loopback-next/blob/typeorm-yaapa/packages/typeorm/src/typeorm.utils.ts.